scripts/trainer.py

```diff
@@ -153,8 +198,32 @@ def parse_args(args):
 def main():
     args = parse_args(sys.argv[1:])
 
-    retriever = Trainer()
-    retriever.go(args)
+    if args.train_with_taskcluster:
+        task_definition = {
+            "payload": {
+                "command": [
+                    "python",
+                    "-m",
+                    "scripts.trainer",
+                    args.model,
+                    "--limit",
+                    str(args.limit),
+                    "--no-download" if not args.download_db else "",
+                    "--download-eval" if args.download_eval else "",
+                    "--lemmatization" if args.lemmatization else "",
+                    "--classifier",
+                    args.classifier,
```

Comment:
I would avoid hard-coping the args list here. We could reuse the original command, just stripping `--taskcluster` from it.

Reply:
I have done this. Please check the latest `command` and let me know.

Reply:
Thank you!